Client/qtTeamTalk/useraccountsmodel.cpp

```diff
@@ -23,6 +23,11 @@ UserAccountsModel::UserAccountsModel(QObject* parent)
 {
 }
 
+void UserAccountsModel::setTableView(QTableView* view)
+{
```

Comment:
The view class should not be passed to the model. Then the model cannot be used by other classes. What you need in the model has to be inside the model.

Reply:
Do you know how to retrieve horizontalHEader or something similar directly inside the model? I don't see how to do it without pass the tableView to the model :(

Reply:
hm, not quite sure. Maybe you have to tell the model of the column index order, since it should be irrelevant to the model.

Reply:
I tried something, but the behaviour is totally broken :(
So, for now, not sure how to do, I have to do more experimentation